.gitignore

```diff
@@ -3,4 +3,4 @@ wabt
 TAGS
 tags
 wasm-spec
-wasi
```

Comment:
I think we should leave this in the gitnore.

Reply:
Need to delete this to make changes to wasi.